components/raftstore/src/store/worker/read.rs

```diff
@@ -1153,6 +1169,47 @@ where
                             }
                         }
                     }
+                    RequestPolicy::ReadIndex => {
+                        TLS_LOCAL_READ_METRICS
+                            .with(|m| m.borrow_mut().local_received_follower_read_requests.inc());
+                        let read_ts_valid = {
+                            if req.get_header().get_flag_data().is_empty() {
+                                false
+                            } else {
+                                let read_ts =
+                                    decode_u64(&mut req.get_header().get_flag_data()).unwrap();
```

Comment:
Better to not call it directly at the upper level, as it will make the code difficult to understand.

And `decode` with `unwrap()` introduces a strong implicit constraint, even though it seems to be set by all the read request types, better to check before using.

Reply:
1. `propose_raft_command` is a high level function, better not to expose the `req` decoding related details in it. For example, wrapping the follower read processing logic into some `try_local_folllower_read` function and call it in `propose_raft_command`.
2. Better not to `unwrap` here but to process the possible error, reducing the implicit, reducing implicit assumption dependencies between code.